### DISCUSSION TITLE
[Question] is there compatibility with SP7000+?

### DISCUSSION THREAD

**Discussion by mrMiimo** [ParadoxAlarmInterface/pai#383] [Q&A] [answered]
Hi,
does anyone know if there is compatibility with SP7000+ ?
I noticed that it is difficult to find a SP7000 panel and on the site the firmware version for the sp7000+ panel is different, for this the currently it is V1.29.000

qualsiasi aiuto è gradito, grazie mille!
Rod.

**Comment by yozik04** [comment, marked as answer]
According to this PR https://github.com/ParadoxAlarmInterface/pai/pull/365 it should be supported in dev branch.
Most probably with IP150. @makstech did not came back to tell us if it worked for him.

**Reply by mrMiimo** [reply]
oh great!!! I sleep more peacefully 😅
I can only find the IP150+ adapter... Do you think I could find problems?
Unfortunately, I can't find the specifications of the differences between paradox hardware versions 🤷🏻‍♂️
Tnks a lot!

**Reply by yozik04** [reply]
I do not think there will be any difference if it is IP150 or IP150+. If you get it working, please update our compatibility table.

**Reply by makstech** [reply]
> @makstech did not came back to tell us if it worked for him.  

But I also didn't say that it doesn't work 😄  
I have IP150+ and it works fine, except for some nuances with voltages, i.e. https://github.com/ParadoxAlarmInterface/pai/issues/366

**Reply by mrMiimo** [reply]
I was thinking the same thing...
**Certainly**! ..it will take a while because I still have to place the order

**Reply by mrMiimo** [reply]
done

**Comment by greggo74** [comment]
I'm new to PAI but now have it working with my SP7000, but it's currently disconnecting frequently ands not reconnecting.  But it definitely works fine when it's connected.
Using IP150 direct connection.

**Reply by yozik04** [reply]
With IP module? With IP module it should be working fine. If your network works fine.
